php/docker/templates/nginx-vhost-conf.d/75-page-cache-locations.conf

```diff
@@ -1,5 +1,9 @@
 # vim: set ft=nginx:
 
+{{- $truthStrings := list "true" "on" "yes" -}}
```

Comment:
`isTrue` should work
https://github.com/jwilder/dockerize#using-templates

Reply:
Nice one! I wasn't aware of it...